people.py

```diff
@@ -22,6 +22,14 @@ def _add_oliver(people):
     key = new_p.first_name + '_' + new_p.surname
     people[key] = new_p
 
+
+def _add_road_runner(people):
+    """ adds oliver """
+    new_p = PERSON(first_name='Road', surname='Runner',
+                   age=71, pet=None, hobbies=['running', 'seed'])
+    key = new_p.first_name + '_' + new_p.surname
+    people[key] = new_p
+
```

Comment:
PEP8 says there should be two blank lines here

Reply:
fixed in 9d6c0ba 

@osmart is this OK?